content/blog/2022/10/kubewarden_1_3_release.md

```diff
@@ -0,0 +1,135 @@
+---
+title: "Kubewarden 1.3 is out!"
+authors:
+- Flavio Castelli
+date: 2022-10-26
+---
+
+The Kubewarden development team is happy to announce the release of the Kubewarden 1.3 stack.
+
+In addition to the usual amount of small fixes, this release focused on the following
+themes.
+
+## Improve end users confidence
+
+We want our users to feel confident about using Kubewarden, knowing that good
+development and security practices are being followed by the Kubewarden project.
+We think this is particularly relevant to Kubewarden, given our users trust us
+to keep their Kubernetes clusters secure and compliant.
+
+To achieve that, we've joined the [CLOMonitor](https://clomonitor.io/) initiative.
+This is a project from the Clound Native Foundation which performs a series of
+checks to ensure certain project health best practices are met.
+
+We are proud to announce that Kubewarden is currently A-rated, with a score above
+the 90%. The results are completely transparent and can be found by visiting
+the [Kubewarden dashboard](https://clomonitor.io/projects/cncf/kubewarden)
+on the CLOMonitor website.
+
+## Performance improvements
+
+We have reduced the startup time of Policy Server, the component in charge of
+running all our WebAssembly policies.
+The difference is quite significant when multiple policies are enforced by the
```

Comment:
would it make sense to mention it was an order of magnitude faster?

Reply:
I don't have the data around anymore :/

Do you feel something like:

> Starting a Policy Server with multiple policies defined is now an order of magnitude faster

would be better?